src/lib/LUA/tx_devLUA.cpp

```diff
@@ -14,6 +14,7 @@ static char tlmBandwidth[] = " (xxxxbps)";
 static const char folderNameSeparator[2] = {' ',':'};
 static const char switchmodeOpts4ch[] = "Wide;Hybrid";
 static const char switchmodeOpts8ch[] = "8ch;16ch Rate/2;12ch Mixed";
+static const char antennamodeOpts[] = "Gemini;Ant 1;Ant 2";
```

Comment:
Could we also have a "Diversity" mode which alternates the chip that is being use for output.

Reply:
...but still use both RX for receiving telemetry. Otherwise simple antenna diversity would be sufficient.

Reply:
indeed

Reply:
Discussed on discord and we can look at this in a future PR.